tests/unit/s2n_ems_extension_test.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+/* For testing static function s2n_server_ems_should_send */
+#include "tls/extensions/s2n_server_ems.c"
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    /* s2n_server_ems_should_send */
```

Comment:
I'm not sure just testing should_send is enough. Maybe some kind of functional test to make sure it's setting `ems_negotiated` properly during the handshake, and not setting it when it shouldn't? That'll make sure you've got the extension all wired up properly.